pkgs/by-name/au/autosuspend/package.nix

```diff
@@ -32,9 +32,9 @@ python3.pkgs.buildPythonApplication rec {
     portalocker
     psutil
     python-dateutil
-    pytz
     requests
     requests-file
+    tzdata
```

Comment:
Note for reviewers: remove pytz, require tzdata

See https://github.com/languitar/autosuspend/releases/tag/v8.0.0